packages/cozy-harvest-lib/src/components/AccountsPaywall/AccountsPaywall.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
Can you add `//@ts-check` at the beginning on every new file you create on harvest please?